server/plugin.go

```diff
@@ -39,6 +40,10 @@ func (p *Plugin) FilterPost(post *model.Post) (*model.Post, string) {
 	for i, word := range words {
 		if p.WordIsBad(word) {
 			if configuration.RejectPosts {
+				p.API.SendEphemeralPost(post.UserId, &model.Post{
+					ChannelId: post.ChannelId,
+					Message:   fmt.Sprintf(configuration.WarningMessage, word),
```

Comment:
What happens if a users tries to post a bad word as response to a thread? Does the ephemeral post get posted in the RHS?

Reply:
Just tested this: the ephemeral post get posted in the RHS. Is that a problem? I don't know if we can post an ephemeral message in the RHS (tested both with `post.RootId` and `post.ParentId` and it doesn't work)

Reply:
Yes, you can post an ephemeral message in the RHS by setting `RootId`. `post.RootId` already contains the thread `RootId`, which you can use.

Reply:
Thanks for the input!